insights/parsers/package_provides_httpd.py

```diff
@@ -0,0 +1,63 @@
+"""
+PackageProvidesHttpd - command ``/bin/echo {httpd_command_package}``
+====================================================================
```

Comment:
Please follow the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) to add this parser to the insights-core documentation.